tests/containers/ReportContainer_spec.js

```diff
@@ -496,4 +497,24 @@ describe('Report Container', () => {
       done();
     }, 5001);
   });
+
+  it('Generate test template layout report show bar values', (done) => {
+    const testTemplate = TemplateProvider.getTestLayoutTemplate();
+    const toRender = <ReportContainer sections={prepareSections(testTemplate)} />;
+    const reportContainer = mount(toRender);
+
+    const reportLayouts = reportContainer.find(ReportLayout);
+    expect(reportLayouts).to.have.length(1);
+
+    setTimeout(() => {
+      const barChart = reportContainer.find(SectionBarChart);
+      expect(barChart).to.have.length(3);
+
+      expect(barChart.at(0).props().chartProperties.showValues).to.equal(undefined);
+      expect(barChart.at(1).props().chartProperties.showValues).to.equal(true);
+      expect(barChart.at(2).props().chartProperties.showValues).to.equal(undefined);
+      expect(barChart.at(1).find('.recharts-label').length).to.equal(2);
```

Comment:
worth validating that the textContent meets expectations (this would cover the values formatting + sum)